src/applications/check-in/README.md

```diff
@@ -12,6 +12,7 @@ Before you get started check [this page](https://depo-platform-documentation.scr
   - clone vets-website repo `git clone git@github.com:department-of-veterans-affairs/vets-website.git`
   - navigate to the check-in application `cd src/applications/check-in`
   - run `yarn install`
+  - run `yarn workspace @department-of-veterans-affairs/applications-check-in install`
   - turn on local mocks `yarn mock-api --responses src/applications/check-in/api/local-mock-api/index.js`
```

Comment:
Unrelated/optional I didn't realize I needed to change back to the root directory.

Reply:
nice little quality of life improvement, I updated the command to work from anywhere in the repo :)